src/scenegraph/Loader.h

```diff
@@ -8,6 +8,7 @@
  */
 #include "BaseLoader.h"
 #include "CollisionGeometry.h"
+#include "Mount.h"
```

Comment:
:sweat: ...after lot of decoupling I end putting this include here without any needs... No needs even for a forward declaration!